services/core-web/package.json

```diff
@@ -44,7 +44,7 @@
     "react": "^16.13.1",
     "react-async-script-loader": "^0.3.0",
     "react-dom": "^16.13.1",
-    "react-filepond": "^7.0.1",
+    "react-filepond": "^7.1.0",
```

Comment:
Have you verified if filepond works correctly after this change? Especially the inspector signature upload?

Reply:
Yep, it works!